acceptance_tests/features/steps/events.py

```diff
@@ -0,0 +1,13 @@
+from behave import step
+
+from acceptance_tests.utilities.event_helper import check_if_event_list_is_exact_match
+
+
+@step("the events logged for the case the respondent has authenticated against are {expected_event_list}")
+@step("the events logged for the case requesting telephone capture are {expected_event_list}")
```

Comment:
I prefer a single step name, something like 'events logged against first emitted case are {expected_event_list}

Feels less misleading to me?

Reply:
This could be left to refactoring work.